script/danger/pnpm-lock.yaml

```diff
@@ -1,620 +1,260 @@
-lockfileVersion: '9.0'
+lockfileVersion: '6.0'
```

Comment:
@SomeoneToIgnore Could you upgrade your pnpm version? The lockfile regressed here.

Reply:
Thank you, should be better now as the version is not changed.